modnet/models.py

```diff
@@ -312,10 +323,17 @@ def fit_preset(
         presets: List[Dict[str, Any]] = None,
         val_fraction: float = 0.15,
         verbose: int = 0,
-        refit=True,
+        classification: bool = False,
+        refit: bool = True,
+        fast: bool = False,
+        nested: bool = False,
```

Comment:
maybe add possibility to choose number of folds

Reply:
done